### PR TITLE
ensure we always can rake

### PR DESCRIPTION
**Pull request by thommay**
cc @danielsdeleo @chef/client-core

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -5,6 +5,8 @@ gem "activesupport", "< 4.0.0", group: :compat_testing, platform: "ruby"
 
 gem "chef-config", path: "chef-config" if File.exist?(__FILE__ + "../chef-config")
 
+gem "rake"
```
Comment (danielsdeleo) [inline Gemfile]: Probably should have a comment to explain that we need this so rake is still available after bundling without the development group.

**Comment by danielsdeleo** [comment]
:+1:

**Comment by lamont-granquist** [comment]
should probably be ripped out of the gemspec or i think bundler whines about being defined twice...

but :+1:

**Comment by thommay** [comment]
@lamont-granquist looks like bundler doesn't whine, so I'm gonna go ahead and merge

**Comment by coderanger** [comment]
Bundler only objects if it's listed in the Gemfile itself twice.

**Pull request merged**